submissions/YarHadi/oop_exercise/index.js

```diff
@@ -0,0 +1,55 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/YarHadi/a-tiny-JS-world
+   Web app: https://yarhadi.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+   name;
+   gender;
+   saying;
```

Comment:
Why this approach to declaring class properties is used only for `Inhabitant` class?
This is not a rhetoric question.